src/pretix/base/services/mail.py

```diff
@@ -455,6 +461,20 @@ def mail_send_task(self, *args, to: List[str], subject: str, body: str, html: st
                         logger.exception('Could not attach invoice to email')
                         pass
 
+        if attach_other_files:
+            for fname in attach_other_files:
+                ftype, _ = mimetypes.guess_type(fname)
+                data = default_storage.open(fname).read()
+                try:
+                    email.attach(
+                        re.sub(r'^(.*)(\.[^.]+)(\.[^.]+)$', r'\1\3', os.path.basename(fname)),
```

Comment:
Why do we need to trim the last part separated by a dot before the file-extensions? This would turn `file.tar.gz` into `file.gz`